pkg/config/config.go

```diff
@@ -522,3 +525,47 @@ func GetSrcType(ctStr string) (SrcType, error) {
 		return NONE, fmt.Errorf("Unsupported component source type: %s", ctStr)
 	}
 }
+
+// CorrectSourcePath corrects the current sourcePath depending on local or binary configuration,
+// if Git has been passed, we simply return the source location from LocalConfig
+// this will get the correct source path whether on Windows, macOS or Linux.
+func CorrectSourcePath(localConfig *LocalConfigInfo) (path string, err error) {
+
+	cmpName := localConfig.GetName()
+	sourceType := localConfig.GetSourceType()
+	sourceLocation := localConfig.GetSourceLocation()
+
+	if sourceLocation == "" {
+		return "", fmt.Errorf("Blank source location provided")
+	}
+
+	// If we are passing in a GIT source type, we make no changes.
+	// if it's binary or local, we will correct the source path.
+	if sourceType == GIT {
+		glog.V(4).Info("Git source type detected, not correcting SourcePath location")
+		return sourceLocation, nil
+	} else if sourceType == BINARY || sourceType == LOCAL {
+
+		// We use "ToSlash" here to make sure that deliminators are in a slash format
+		sourcePath := filepath.ToSlash(localConfig.GetSourceLocation())
```

Comment:
Shouldn't it be `FromSlash`?

Reply:
I tried it with FromSlash and it does not work with Windows. I have a feeling it's because Go / Windows works by default with forward slashes. Using "ToSlash" will work regardless since if the user passes in C:\foo\bar it'll convert to C:/foo/bar anyways.

Reply:
I thought that this function is used for reading.
Full windows path like `C:\foo\bar` should never get into the config file